help/custom-profile-fields.md

```diff
@@ -78,7 +78,7 @@ checkboxes will be disabled.
 
 1. Toggle **Display on user card**.
 
-4. Click **Save changes**.
+1. Click **Save changes**.
```

Comment:
This can be changed to follow consistency but should be done in a separate commit.